bgpd/bgp_open.h

```diff
@@ -24,7 +24,7 @@
 /* Standard header for capability TLV */
 struct capability_header {
 	uint8_t code;
-	uint8_t length;
+	uint8_t length; /* Note, extra one more oct in dynamic capabilities */
```

Comment:
what 'oct' means here?

Reply:
FRR implemented the old version of dynamic capability draft, which has only 1 Octect for "length field".  This field has been changed in the latest version to 2 Octects. Since the draft hasn't become RFC, and by talking to the authors, they are still trying to finalize the content, hence I didn't change the code, but left some comment there.

Reply:
Since it's still a draft, what bgp implementations supports it? Why SONiC needs to support it?

Reply:
Almost all switch vendors with BGP implementation support dynamic capability - Cisco, Juniper, Arista, Huawei...